src/components/dev-hub/card.js

```diff
@@ -127,8 +110,10 @@ const Card = ({
         >
             <div>
                 {!collapseImage && (
-                    <ImageWrapper>
+                    <ImageWrapper placeChildrenOnImage={placeChildrenOnImage}>
                         {image && <Image src={image} alt="" />}
+                        {/* Expect children to be absolutely positioned on the image */}
+                        {placeChildrenOnImage && children}
```

Comment:
Here is where we toggle where children are rendered. Another note is that we don't really use `children` anywhere as-is so this bool isn't 100% necessary, however since it isn't the typical card-child behavior it might be worth keeping.

Reply:
This is being used to insert the video modal component inside the imagewrapper?

Reply:
This is where the video button is being placed inside the image wrapper. Another idea would just be to move the video button here and toggle it with a prop actually, how does that sound?

(That way `children` still behaves consistently)